Homework/Van_Le/lesson11/ThreeSumsSmaller.cs

```diff
@@ -0,0 +1,60 @@
+﻿namespace LeetCodeSolutions
+{
+
+    /// <summary>
+    /// 
+    /// First, to make it easy to use two pointers efficiently, this array needs to be sorted.
+    /// 
+    /// Use two pointers with left one which points to the first item of the array
```

Comment:
good job, it is good way to practice how to explain to interviewer. You can also try to explain verbally what you wrote as well.